src/OmniSharp.MSBuild/Resolution/PackageDependencyResolver.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using NuGet.ProjectModel;
+using OmniSharp.MSBuild.ProjectFile;
+
+namespace OmniSharp.MSBuild.Resolution
+{
+    internal class PackageDependencyResolver
```

Comment:
Added https://github.com/OmniSharp/omnisharp-roslyn/issues/863 to track adding tests.